nifi-nar-bundles/nifi-hive-bundle/nifi-hive-processors/src/test/java/org/apache/nifi/processors/hive/TestPutHiveQL.java

```diff
@@ -42,25 +41,22 @@
 
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertThrows;
 import static org.junit.Assert.assertTrue;
-import static org.junit.Assert.fail;
 
 public class TestPutHiveQL {
     private static final String createPersons = "CREATE TABLE PERSONS (id integer primary key, name varchar(100), code integer)";
     private static final String createPersonsAutoId = "CREATE TABLE PERSONS (id INTEGER NOT NULL GENERATED ALWAYS AS IDENTITY (START WITH 1), name VARCHAR(100), code INTEGER check(code <= 100))";
 
-    @Rule
-    public TemporaryFolder folder = new TemporaryFolder();
```

Comment:
Replacing this approach with `Files.createTempDir()` in individual methods results in a number of directories that need to be cleaned up. Can this approach be changed to use the JUnit 5 `TempDir` annotation so that the use of `Files.createTempDir()` can be removed?